deploy/rp-production.json

```diff
@@ -236,6 +236,49 @@
                 "[resourceId('Microsoft.Network/loadBalancers', 'rp-lb')]"
             ]
         },
+        {
+            "properties": {
+                "severity": 4,
+                "enabled": true,
+                "scopes": [
+                    "[resourceId('Microsoft.Network/loadBalancers', 'rp-lb')]"
+                ],
+                "evaluationFrequency": "PT5M",
+                "windowSize": "PT1H",
+                "targetResourceType": "Microsoft.Network/loadBalancers",
+                "criteria": {
+                    "allOf": [
+                        {
+                            "operator": "LessThan",
+                            "threshold": 99.95,
+                            "AdditionalProperties": null,
+                            "name": "HealthProbeBelow9995",
+                            "metricName": "DipAvailability",
+                            "metricNamespace": "microsoft.network/loadbalancers",
+                            "timeAggregation": "Average",
+                            "criterionType": "StaticThresholdCriterion"
+                        }
+                    ],
+                    "AdditionalProperties": null,
+                    "odata.type": "Microsoft.Azure.Monitor.SingleResourceMultipleMetricCriteria"
+                },
+                "autoMitigate": true,
+                "actions": [
+                    {
+                        "actionGroupId": "[resourceId('Microsoft.Insights/ActionGroups', 'rp-lb-health-ag')]",
```

Comment:
Surely you need to refer to the name of the resource group containing rphealth-ag in [resourceId](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#resourceid).  Also rp-lb-health-ag is a typo.

Reply:
Moved to a template parameter.